lizmap/modules/view/controllers/translate.classic.php

```diff
@@ -23,6 +23,7 @@ public function index()
         $rep = $this->getResponse('binary');
         $rep->mimeType = 'text/javascript';
         $rep->doDownload = false;
+        $rep->setExpires('+1 year');
```

Comment:
on year is too much and useless. Few days max. Few hours is good imho.

Reply:
We invalidate cache when needed so we can put a very long value. It is a [best practice](https://web.dev/uses-long-cache-ttl/) : 

> When possible, cache immutable static assets for a long time, such as a year or longer.

Reply:
>We invalidate cache

how?

Reply:
As we change LWC version parameter in url when we update LWC, the new version of translation file is loaded.
```php
$version = (string) $xmlLoad->info->version;
$rep->addJSLink((jUrl::get('view~translate:index')).'?v='.$version.'&lang='.jLocale::getCurrentLang().'_'.jLocale::getCurrentCountry());`